source/sgp_mode/SGPWorld.h

```diff
@@ -94,9 +94,17 @@ class SGPWorld : public SymWorld {
     }
 
     scheduler.ProcessOrgs([&](emp::WorldPosition pos, Organism &org) {
-      org.Process(pos);
-      if (org.GetDead()) { // Check if the host died
-        DoDeath(pos);
+      if (org.IsHost()) {
+        org.Process(pos);
+        if (org.GetDead()) DoDeath(pos);
+      }
+      else {
+        //have to check for death first, because it might have moved
+       // process takes worldposition, dosymdeath takes popid
+        if (org.GetDead()) DoSymDeath(pos.GetPopID());
```

Comment:
We should probably look into how to hook up to the death signal to handle checking for Symbionts there, or use the SetKillOrgFun functionality to make our own custom death handling, but can be a future task